apps/api/src/app/inversify.config.ts

```diff
@@ -21,6 +24,7 @@ import {
   slippageServiceSymbol,
   usdServiceSymbol,
 } from '@cowprotocol/services';
+import ms from 'ms';
```

Comment:
Nitpick: import sorting. `ms` is imported after the const using it is defined.
Better to move the const down.

Reply:
I think we should make it automatic the sorting and cleanup of unused imports. Making an issue for this https://github.com/cowprotocol/bff/issues/76